service/.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hi @Braimah101 I think this file needs to be untracked and removed from your changes.